src/blocks/block-coauthor-image/block.json

```diff
@@ -0,0 +1,66 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "co-authors-plus/image",
+	"version": "1.0.0",
+	"title": "Co-Author Featured Image",
+	"category": "theme",
+	"description": "Uses your theme's image sizes to display a scalable avatar for a co-author with a guest author profile. Does not fallback to Gravatar images.",
+	"keywords": [ "coauthors" ],
+	"supports": {
+		"align": [ "left", "right", "center", "wide", "full" ],
+		"__experimentalBorder": {
+			"color": true,
+			"radius": true,
+			"width": true,
+			"__experimentalSelector": "img, .block-editor-media-placeholder",
+			"__experimentalSkipSerialization": true,
+			"__experimentalDefaultControls": {
+				"color": false,
+				"radius": false,
+				"width": false
+			}
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true,
+			"__experimentalDefaultControls": {
+				"margin": false,
+				"padding": false
+			}
+		}
+	},
+	"usesContext": [ "co-authors-plus/author" ],
+	"attributes": {
+		"isLink": {
+			"type": "boolean",
+			"default": false
+		},
+		"rel": {
+			"type": "string"
+		},
+		"aspectRatio": {
+			"type": "string"
+		},
+		"width": {
```

Comment:
I see your point. I tried to only use sizes that are available in the theme, but when no resolution selection has been made, it picks the last image size in the array, which is often `full` size. Instead it could default to the first item, which is more likely to be `thumbnail`. Would that work?

Reply:
I updated the default size in an attempt to address this here https://github.com/Automattic/Co-Authors-Plus/pull/997/commits/2de42ace3db399ba0f0dd384f44b9503232bbb3f

Reply:
I think this works well -- thanks!